lib/shopify-cli/context.rb

```diff
@@ -336,7 +336,11 @@ def uname
     #   stat = @ctx.system('ls', 'a_folder')
     #
     def system(*args, **kwargs)
-      CLI::Kit::System.system(*args, env: @env, **kwargs)
+      process_status = CLI::Kit::System.system(*args, env: @env, **kwargs)
+      unless process_status.success?
+        abort("System call failed: #{args.join(' ')}")
+      end
+      true
```

Comment:
Another approach here would have been to return `process_status.success` and have the caller responsible for checking the result, but I feel this way is safer.

Reply:
I agree, if we can rescue the `abort` call it's safer to always fail unless explicitly handling errors.